data/core/units.cfg

```diff
@@ -247,7 +247,24 @@ The Dunefolk’s inquisitive and explorative nature does not preclude military s
+        description= "Saurians are small, slender, and lizard-like creatures. Much smaller than humans, they rarely stand taller than a ten year old child, though the distance from the tip of their snout to the end their tail can be as long as the average man is tall. Light and nimble, the warriors prefer to fight as they hunt- slipping through enemy lines to target the weak and the injured while their slender figures allow them to evade their attackers.
+
+<header>text='Society'</header>
+Saurians are very mysterious creatures due to their tendency to live in areas inhospitable to others, such as swamps. Fatalistic in the extreme, Saurians believe all the events in a life can be predicted by the use of a complex form of astrology.
+
+Saurian culture is sharply segregated between the genders. Within each gender the members compete and through skill, determination, and reputation establish a clear pecking order, with a chief at the top. On those occasions when the two genders interact they do not contest for dominance, the situation determines the dominant gender. The chief of the males is alpha if they are within their village or encampment while the chief of females is dominant anywhere else. This continues down the rank structure with each male or female being dominant over any member of the opposite gender with lower rank and submitting to members of the opposite gender with higher rank.
+
+The segregation and alternating gender-dominance of Saurian society is an outgrowth of their clearly defined gender roles. It is the responsibility of the female to hunt and find food, skills which ultimately train them to be warriors; the skirmishers, flankers, and ambushers other races so fear. Males, meanwhile, are responsible for guarding the clutch- the eggs left by the females. While this leaves time for the males to develop and hone the arts of astrology, healing, and magic, it also exposes them to significant danger, as they are stationary targets for a Saurian clan's number one enemy- other Saurian clans.
+
+New Saurian clans are started when the proper astrological signs are read. Called a “hatching,” the female indicated by the conjunction select a group of individuals with lower rank and leave their source clan. Frequently all females with a specific trait will be selected, causing multiple clans to “hatch” at the same time. Selection is a simple process, no group leaving can be larger than any other, all the groups together cannot be larger, than the group being left, and higher ranking allows a female to overrule another female's choice of who they take.
+
+Because of the rapid growth of saurian populations, these frequent splits in clans, and the fact that cannibalism is not taboo among Saurians violence is one of the defining features of the Saurian life. This limits the growth of the Saurian culture, limiting it to fits and starts- as much of their knowledge is passed by oral tradition and their possessions must be mobile.
```

Comment:
Missing a comma before "violence"? Also drop this entire part: ", limiting it " and replace that hyphen with a comma.